### ISSUE TITLE
Failing Test Jetty 11 - ServletContextHandlerTest.testAsyncServletRequestListenerListener

### ISSUE THREAD

**Issue by olamy** [eclipse/jetty.project#7777]
Jetty 11 build keep failing with this
```
java.lang.AssertionError: 

Expected: <2>
     but: was <9>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at org.eclipse.jetty.servlet@11.0.9-SNAPSHOT/org.eclipse.jetty.servlet.ServletContextHandlerTest.testAsyncServletRequestListenerListener(ServletContextHandlerTest.java:1052)
```

https://jenkins.webtide.net/job/jetty.project/job/jetty-11.0.x/1026/testReport/

**Comment by joakime** [comment]
Should be looked at before we cut the Jetty 11.0.9 release?

**Comment by joakime** [comment]
This isn't a flaky test, it's failing 100% of the time.

**Comment by joakime** [comment]
Fixed in `jetty-11.0.x` (where it was failing)
